diffenent to reference
DNA sequencing 
RNA-seq 
ChIP-seq 
ATAC-seq

bulk 
single-cell transcriptome

Why is RNA-seq read mapping hard? 
errors in the reads (mismatches, insertions, deletions)
errors in the reference
genetic variation
spliced mapping reads contains sequences in 2exons
multi-mapping
efficiency
columns are samples
rows are genes
> Written with [StackEdit](https://stackedit.io/).
<!--stackedit_data:
eyJoaXN0b3J5IjpbMjE1MjgxMzA1LC02OTY4Nzg0MjIsNzMwOT
k4MTE2XX0=
-->