diffenent to reference
DNA sequencing 
RNA-seq 
ChIP-seq 
ATAC-seq

bulk 
single-cell transcriptome

Why is RNA-seq read mapping hard? 
errors in the reads (mismatches, insertions, deletions)
errors in the reference
genetic variation
spliced mapping reads contains sequences in 2exons
multi-mapping
efficiency
columns are samples
rows are genes

each condition multiple samples 
make assumption about the structure of the data
dot is a gene
low variance - artifacts
reduce false positive maximize statistical power

overlap between two set of genes in random genes

some reads just map to one genome
some reads map to two genomes but better 

> Written with [StackEdit](https://stackedit.io/).

UMI unique molecular identifier
10x company offers single cell sequencing

significant difference
pvalue for 

cell cycle effect
a lot genes change expression in cell cycle

order cell along pseudotime based on expression

mRNA transcribed and degradated
<!--stackedit_data:
eyJoaXN0b3J5IjpbLTEyNzIyODE0OTMsLTkwMDg4MzE2MCwtMj
A4ODM0MTE3MiwxMzQ5NDc4MDE2LDE2MjQ5NTA5NzUsLTY1MzMy
NzU0LDgyNDIwNzczOCwtMjE0MzcxNTEwMCwyMTUyODEzMDUsLT
Y5Njg3ODQyMiw3MzA5OTgxMTZdfQ==
-->