diffenent to reference
DNA sequencing 
RNA-seq 
ChIP-seq 
ATAC-seq

bulk 
single-cell transcriptome

Why is RNA-seq read mapping hard? 
errors in the reads (mismatches, insertions, deletions)
errors in the reference
genetic variation
spliced mapping reads contains sequences in 2exons
multi-mapping
efficiency
columns are samples
rows are genes

each condition multiple samples 
make assumption about the structure of the data
dot is a gene
low variance - artifacts
reduce false positive maximize statistical power

overlap between two set of genes in random genes

some reads just map to one genome
some reads map to two genomes but better 

> Written with [StackEdit](https://stackedit.io/).

UMI unique molecular identifier
10x company offers single cell sequencing

significant difference
pvalue for 

cell cycle effect
a lot genes change expression in cell cycle

order cell along pseudotime based on expression
<!--stackedit_data:
eyJoaXN0b3J5IjpbMTUwMjkxMTExNCwtOTAwODgzMTYwLC0yMD
g4MzQxMTcyLDEzNDk0NzgwMTYsMTYyNDk1MDk3NSwtNjUzMzI3
NTQsODI0MjA3NzM4LC0yMTQzNzE1MTAwLDIxNTI4MTMwNSwtNj
k2ODc4NDIyLDczMDk5ODExNl19
-->